src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/config/AuthConfig.java

```diff
@@ -9,7 +9,6 @@
 
 @Configuration
 public class AuthConfig {
```

Comment:
I think below bean config is missing ?

```
@Bean
    @ConditionalOnProperty(name = "idam.s2s-auth.url")
    public AuthTokenValidator tokenValidator(ServiceAuthorisationApi s2sApi) {
        return new ServiceAuthTokenValidator(s2sApi);
    }
```

Reply:
If you rebase and accept the master version of this file, it should be fine.

Reply:
@lgonczar I still have to add the bean config as @nitinprabhu mentioned above. We need this bean to initiallise `AuthTokenValidator` in `AuthService.java`